app/controllers/providers/address_selections_controller.rb

```diff
@@ -46,12 +46,16 @@ def build_addresses_from_form_data
       end
     end
 
+    def address_titleize(sentence)
+      sentence.to_s&.split(' ')&.map(&:capitalize)&.join(' ')
+    end
+
```

Comment:
I think the Rails helper method `#titleize` does this.

if you want to handle `nil`s then:

```sentence&.titleize || ''```

Reply:
`sentence.to_s&.split(' ')&.map(&:capitalize)&.join(' ')` handles hyphens better than `titleize` - `titleize` replaces them with a space, which can make addresses look odd.

Instead, renamed `address_titleize` to `hyphen_safe_titleize` to better describe what it does.